CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.3.0
+
+### Flow
+
+This version is the first stable release with [`Flow`](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/-flow/index.html) API.
+
+All `Flow` API not marked with `@FlowPreview` or `@ExperimentalCoroutinesApi` annotations is stable and here to stay.
+Flow declarations marked with `@ExperimentalCoroutinesApi` has [the same guarantees](/docs/compatibility.md#experimental-api) as regular experimental API.
```

Comment:
"declarations ... has" -> "declarations ... have". I'd also replace "API ... is" in the previous sentence with "APIs ... are stable"